tests/xargs_tests.rs

```diff
@@ -386,3 +426,65 @@ fn xargs_zero_lines() {
         .stderr(predicate::str::contains("Value must be > 0, not: 0"))
         .stdout(predicate::str::is_empty());
 }
+
+#[test]
+fn xargs_replace() {
+    Command::cargo_bin("xargs")
+        .expect("found binary")
+        .args(["-i={}", "echo", "{} bar"])
+        .write_stdin("foo")
+        .assert()
+        .stdout(predicate::str::contains("foo bar"));
+
+    Command::cargo_bin("xargs")
+        .expect("found binary")
+        .args(["-i=_", "echo", "_ bar"])
+        .write_stdin("foo")
+        .assert()
+        .stdout(predicate::str::contains("foo bar"));
+
+    Command::cargo_bin("xargs")
+        .expect("found binary")
+        .args(["--replace=_", "echo", "_ _ bar"])
+        .write_stdin("foo")
+        .assert()
+        .stdout(predicate::str::contains("foo foo bar"));
+
+    Command::cargo_bin("xargs")
+        .expect("found binary")
+        .args(["-i=_", "echo", "_ _ bar"])
+        .write_stdin("foo")
+        .assert()
+        .stdout(predicate::str::contains("foo foo bar"));
+
+    Command::cargo_bin("xargs")
+        .expect("found binary")
+        .args(["-i", "echo", "{} {} bar"])
+        .write_stdin("foo")
+        .assert()
+        .stdout(predicate::str::contains("foo foo bar"));
+
+    Command::cargo_bin("xargs")
+        .expect("found binary")
+        .args(["-I={}", "echo", "{} bar {}"])
+        .write_stdin("foo")
+        .assert()
+        .stdout(predicate::str::contains("foo bar foo"));
+
+    // Combine the two options to see which one wins
+    Command::cargo_bin("xargs")
+        .expect("found binary")
+        .args(["-I=_", "-i", "echo", "{} bar {}"])
+        .write_stdin("foo")
+        .assert()
+        .stdout(predicate::str::contains("foo bar foo"));
```

Comment:
This only tests the case with `-i` winning. However, if you swap the positions of `-I=_` and `-i`, `-I=_` should win, which currently is not the case (see also my previous comment about `override`: https://github.com/uutils/findutils/pull/323/files#r1547994855).